src/components/App.js

```diff
@@ -1,19 +1,21 @@
 import React from 'react';
 import { Switch, Route } from 'react-router-dom';
-import HelloDahlia from './DummyComponent/DummyComponent';
-import style from './app.scss';
+import Navbar from './layout/Navbar';
+import LandingPage from './LandingPage/LandingPage';
+import Footer from './layout/Footer/Footer';
+import './app.scss';
 
 const Welcome = () => (
```

Comment:
Can you put this in a seperate component?

Reply:
Noted, Done